assets/js/app.js

```diff
@@ -39,6 +43,18 @@
             _paq.push(['trackEvent', action, origin, target]);
         }
 
+        function trackOrganization(organization, action, dataset, format) {
+            //todo track view when dataset page is rendered
+            customDimensionDataset = {};
+            customDimensionDataset['dimension'+customDimensionActionDatasetId] = organization;
```

Comment:
Where does the `customDimensionActionDatasetId` variable come from?

Reply:
Initialized in https://github.com/opendata-swiss/ckanext-switzerland/pull/167/files

Reply:
I see, but then please make sure that they are initialized before using them.

Reply:
OK, We load the app.js after the variables in https://github.com/opendata-swiss/ckanext-switzerland/pull/167/files/ed1948c7c655da90f72243067b929a9d7cfabc38#diff-86553814cbab74f96827be9c81acea6d
is that sufficient?

Reply:
Not really since the app.js is also used in WordPress where these variables are never initialized.